code/obj/item/clothing/gloves.dm

```diff
@@ -306,11 +321,18 @@ var/list/glove_IDs = new/list() //Global list of all gloves. Identical to Cogwer
 	material_prints = "insulative fibers, electrically charged"
 	stunready = 1
 	can_be_charged = 1
-	uses = 5
-	max_uses = 5
+	uses = 10
+	max_uses = 10
 	setupProperties()
 		..()
 		setProperty("conductivity", 0)
+	New()
+		..()
+		src.specialoverride = new /datum/item_special/spark()
+		src.specialoverride.master = src
+		src.overridespecial = 1
```

Comment:
possibly wrap this 'set special override' functionality into a proc on all gloves, seems to happen in a bunch of different places